bokeh/io.py

```diff
@@ -609,3 +610,31 @@ def _remove_roots(subplots):
     for sub in subplots:
         if sub in doc.roots:
             doc.remove_root(sub)
+
+def server_cell(server, script):
```

Comment:
This can probably be private, or if it is public, `bokeh.embed` is maybe a better place